pkgs/development/python-modules/vunit/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  pytestCheckHook,
+  colorama,
+  ghdl,
+  ...
+}:
+buildPythonPackage rec {
+  pname = "vunit_hdl";
+  version = "4.7.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-ol+5kbq9LqhRlm4NvcX02PZJqz5lDjASmDsp/V0Y8i0=";
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    colorama
```

Comment:
`colorama` should go to `dependencies`.
https://github.com/VUnit/vunit/blob/80bba0551d8b9df2c47ca087096ae6a7ae30e9d5/setup.py#L81